pulp_rpm/app/serializers.py

```diff
@@ -1103,3 +1104,15 @@ class Meta:
             'data_type', 'checksum_type', 'checksum'
         )
         model = RepoMetadataFile
+
+
+class RpmRepositorySyncURLSerializer(RepositorySyncURLSerializer):
+    """
+    A serializer for Repository Sync.
+    """
+
+    skip = serializers.ListField(
+        help_text=_("List of content types to skip during sync."),
+        required=False,
+        default=[]
+    )
```

Comment:
Shouldn't you add at the end of this serializer the 'class Meta', adding the fields from the inherited serializer?

Reply:
Didn't think so as class I inherited from doesn't have them. Yet when I add Meta class I got an exception (no Meta class).